File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/Java/program5.java

import java.io.*;

class Program {
	public static void main(String[] args) throws IOException{
		int n = 0, k = 1;
		int sum = 0;
		System.out.printf("Enter Number : ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		try{
			n = Integer.parseInt(br.readLine());
		}
		catch(NumberFormatException ne){
			System.exit(0);
		}

		if(n < 0)
			System.exit(0);
		for(int i = 1; i <= n; i++){
			sum = sum + k * 10;
			System.out.printf("%d %c ", k * 10 - 1, (i != n ? '+':'\0'));
			k = k * 10;
		}
		System.out.printf("= %d\n", sum - n);
	}
}

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/Python/program5.py

try:
	n = int(input("Enter Number : "))
except ValueError as e:
	exit(0)
k = 1
sum = 0
if(n < 0):
	exit(0)
for i in range(1,n+1):
	sum = sum + k * 10
	if(i == n):
		print(k * 10 - 1, end = '')
	else:
		print(k * 10 - 1, '+', end = '')
	k = k * 10
	
print("=", sum - n)
